scripts/utils/sign_and_send.js

```diff
@@ -35,7 +35,8 @@ module.exports = function (web3 = web3, artifacts = artifacts) {
     // The value returned by requiredTxGas is encoded in a revert error message. For retrieving the hex
     // encoded uint value the first 68 bytes of the error message need to be removed.
     const txGasEstimate = parseInt(estimateResponse.substring(138), 16)
-    return txGasEstimate
+    // Multiply with 64/63 due to EIP-150
```

Comment:
Could we put a link to this EIP here?

https://github.com/ethereum/EIPs/blob/master/EIPS/eip-150.md